### ISSUE TITLE
CI detection not working entirely on GitHub actions in 14.0.1

### ISSUE THREAD

**Issue by TheRealWaldo** [release-it/release-it#699]
Everything was working in 13.7.1; but the latest changes seem to not detect it's CI, so prompts are occurring:

```
  ! git fetch
  $ git describe --tags --abbrev=0  [cached]
  $ git symbolic-ref HEAD
  $ git for-each-ref --format="%(upstream:short)" refs/heads/dependabot/npm_and_yarn/release-it-14.0.1
  
  🚀 Let's release gh-action-release-it (currently at 0.1.1)
  
  
  Changelog:
  ### [0.1.2](https://github.com/TheRealWaldo/release-it/compare/v0.1.1...v0.1.2) (2020-09-04)
  
  $ Writing version to README.md
  ! npm version 0.1.2 --no-git-tag-version
  ! npm run lint-fix && npm run build
  $ Writing changelog to undefined
  $ git status --short --untracked-files=no
  
  Changeset:
  M README.md
   M dist/index.js
  
  ! git add . --update
  ? Commit (chore(release): v0.1.2)? (Y/n) 
```

**Comment by webpro** [comment]
Sorry @TheRealWaldo, I forgot to change the import when I switched back from `ci-info` to `is-ci. I've added a test and a fix for this in [v14.0.2](https://github.com/release-it/release-it/releases/tag/14.0.2)

**Comment by TheRealWaldo** [comment]
Looks like it’s working!  Thanks again @webpro !
